src/SFA.DAS.EAS.Web/Views/EmployerTeam/Index.cshtml

```diff
@@ -137,7 +138,30 @@ else
                                                     @task.ItemsDueCount cohort request@(task.ItemsDueCount > 1 ? "s" : "") ready for approval <a href="@Url.EmployerCommitmentsAction("apprentices/cohorts/review")">View cohorts</a>
                                                 </p>
                                                 break;
-                                        }
+                                            case "ReviewConnectionRequest":
+                                                <p>
+                                                    @{
+                                                        var message = "";
+                                                        var action = "";
+                                                        var linkText = "";
+                                                        if (task.ItemsDueCount == 1)
+                                                        {
+                                                            message = "1 connection request to review";
+                                                            action = Url.Action("Outstanding", "TransferConnectionInvitations");
+                                                            linkText = "View Details";
+                                                        }
+                                                        else
+                                                        {
+                                                            message = $"{task.ItemsDueCount} requests to review";
+                                                            action = Url.Action("Index", "Transfers");
```

Comment:
can we change the message to "{task.ItemsDueCount} connection requests to review"